app/assets/stylesheets/recipe_new.scss

```diff
@@ -0,0 +1,102 @@
+@import "colors";
+
+.recipe-name-input-block textarea {
+  font-size: 16pt;
+  font-family: 'Lobster', cursive;
+  background-color: $white;
+  color: $slate;
+  border-color: darken($light-yellow, 5%);
+  border-radius: 5px;
+}
+
+.recipe-attributes-block {
+  padding: 0px 5px;
+  border-radius: 5px;
+  background-color: $dark-orange;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);
+  font-size: 14pt;
+  padding-top: 10px;
+}
+
+.recipe-attributes-block input {
+  background-color: $white;
+  color: $slate;
+  border-color: darken($dark-orange, 5%);
+  border-radius: 5px;
+}
+
+.recipe-attribute-input label {
+  font-size: 10pt;
+}
+
+.recipe-add-categories-block {
+  padding: 0px 5px;
+  border-radius: 5px;
+  background-color: $light-orange;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);;
+  font-size: 14pt;
+  padding-top: 15px;
+  margin: 20px 0;
+}
+
+.recipe-add-categories-block select {
+  background-color: $white;
+  color: $slate;
+  border-color: darken($light-orange, 5%);
+  border-radius: 5px;
+}
+
+.recipe-add-ingredients-block {
+  padding: 0px 5px;
+  border-radius: 5px;
+  background-color: $light-green;
+  box-shadow: 2px 2px 1px darken($light-yellow, 70%);;
+  font-size: 14pt;
+  padding-top: 10px;
+  margin-bottom: 20px;
+}
+
+.recipe-add-ingredients-block select {
+  background-color: $white;
+  color: $slate;
+  border-color: darken($light-green, 5%);
+  border-radius: 5px;
+}
+
+.recipe-add-ingredients-block input {
+  background-color: $white;
+  color: $slate;
+  border-color: darken($light-green, 10%);
+  border-radius: 5px;
+}
+
+.recipe-add-steps-block {
+  padding: 0px 15px;
```

Comment:
Properties should be ordered background-color, border-radius, box-shadow, font-size, padding, padding-top<br>`0px` should be written without units as `0`